client/icecc-create-env.in

```diff
@@ -485,7 +489,8 @@ if test -n "$clang"; then
     # HACK: Clang4.0 and later access /proc/cpuinfo and report an error when they fail
     # to find it, even if they use a fallback mechanism, making the error useless
     # (at least in this case). Since the file is not really needed, create a fake one.
-    if test -d /proc; then
+    # Only add an empty file if the user did not attempt to add there own.
```

Comment:
I am a bit suspicious of the need to "include" files that don't exist on the source system to satisfy running a binary on a target system. I think we want a more explicit "create a fake file" mechanism.